cognite/client/data_classes/data_modeling/data_types.py

```diff
@@ -73,7 +73,7 @@ def _load(cls, resource: dict[str, Any], cognite_client: CogniteClient | None =
         type_ = resource["type"]
         obj: Any
         if type_ == "text":
-            obj = Text(is_list=resource["list"], collation=resource["collation"])
+            obj = Text(is_list=resource["list"], collation=resource.get("collation", "ucs_basic"))
```

Comment:
**issue:** `is_list` should also be optional, and the same applies to all the other types. Also wouldn't it be best to not set the default value in the SDK and just rely on the API to set the default?

Reply:
Nevermind "ucs_basic" is already defined as the default type for collation in `Text` so disregard the original comment but don't need to use that here